packages/validator/src/cli-validator/utils/print-results.js

```diff
@@ -7,13 +7,10 @@ const each = require('lodash/each');
 const pad = require('pad');
 
 // this function prints all of the output
-module.exports = function print(
-  logger,
-  results,
-  chalk,
-  summaryOnly,
-  errorsOnly
-) {
+module.exports = function print({ chalk, config, logger }, results) {
+  const summaryOnly = config.summaryOnly;
+  const errorsOnly = config.errorsOnly;
```

Comment:
Another small style suggestion - I think you can do:
```js
const { summaryOnly, errorsOnly } = config;
```

if you want, but obviously not a requirement.

Reply:
done